### ISSUE TITLE
Cannot specify an empty array for Tips

### ISSUE THREAD

**Issue by lokka30** [piqueserver/piqueserver#623]
**Describe the bug**
Can't set `tips = []` in `config.toml`

**To Reproduce**
N/A

**Expected behavior**
It should disable tips entirely, since there is an empty array.

**Stack Trace or logs**
```
[stderr#error] Traceback (most recent call last):
[stderr#error]   File "/usr/lib/python3.8/asyncio/events.py", line 81, in _run
[stderr#error]     self._context.run(self._callback, *self._args)
[stderr#error]   File "/usr/lib/python3/dist-packages/twisted/internet/asyncioreactor.py", line 290, in run
[stderr#error]     f(*args, **kwargs)
[stderr#error]   File "/usr/local/lib/python3.8/dist-packages/piqueserver/server.py", line 850, in send_tip
[stderr#error]     line = self.tips[random.randrange(len(self.tips))]
[stderr#error]   File "/usr/lib/python3.8/random.py", line 216, in randrange
[stderr#error]     raise ValueError("empty range for randrange()")
[stderr#error] ValueError: empty range for randrange()
```

**Server (please complete the following information):**
 - OS/Version: Ubuntu
 - Piqueserver Version: Latest from repo
 - Installation Method: git

**Client (if relevant)**
N/A

**Additional context**
Running a test server where I don't need tips to be announced.

**Comment by Jipok** [comment]
Very strange. This code should not be called with empty tips. Are you sure you are using the latest version from the repository?
Can not reproduce.

**Comment by NotAFile** [comment]
I think this could happen when using config reloading was that the case?

**Comment by lokka30** [comment]
> Piqueserver Version: Latest from repo
As of Apr 25, main branch

> Config reloading
I always completely restart my piqueserver instances with config changes - I am not aware of any reloading solutions other than passreload which only reloads passwords afaik.

It should be reproducable by setting `tips = []` in config.toml - my intent was to disable tips entirely. I haven't tried with the *current* version in the main branch but I assume it could still be prevalent since this issue hasn't received any comments whatsoever until recently

**Comment by Jipok** [comment]
I tried this: 
```
git clone https://github.com/piqueserver/piqueserver
cd piqueserver
python3 -m venv venv
source venv/bin/activate
pip install -r requirements.txt
python setup.py install
piqueserver --copy-config -d home
# Change tips = [] and tips_frequency = "1s"
piqueserver -d home
```
I do not confirm the problem.
Python 3.9.5  
gcc 10.2.1 20201203

**Comment by Jipok** [comment]
> I haven't tried with the current version in the main branch but I assume it could still be prevalent since this issue hasn't received any comments whatsoever until recently

This is discouraging. At the beginning of the topic, you wrote that you have the latest version from the repo. 
I remember this problem. And apparently I fixed it. Although right now I don't understand why such a change was made, but here is the commit: https://github.com/piqueserver/piqueserver/commit/b12e7e6c01296604723f0f2bec9677570112b371

**Comment by lokka30** [comment]
Unless this is proven fixed then I don't believe closing this issue is an appropriate idea.

> And apparently I fixed it.
What is for certain, on April 25th I experienced this issue. If it is going to be closed because it was 'fixed' previously, then that's okay, but my only intention was to help squash bugs on this platform, not to create an issue out of the blue running an ancient version of piqueserver. The point is that I received errors when adjusting the setting in the manner I described, so as of Apr 25th, it wasn't actually fixed. I created my server a few days before February began (Jan 25th if I remember correctly), running off the main branch's latest files at the time. Of course I updated the server over time, quite infrequently however.

Just to reiterate, this issue was posted on April 25. As there were no comments on this issue until a few days ago, it is absolutely possible that things have changed since, and the version that I was using is no longer able to be considered 'latest'. I can guarantee that the server was at most up to 3 months old. Something I can't guarnatee is that I do think I was working on a 'test server' at the time which would have been running a very recent version of piqueserver.

I had transferred ownership of my server months ago so I am unsure if it is currently an issue or not. I don't have a VPS to test this on either, otherwise I would have replied with my results.
